<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetstOperateLogClass</class>
 <widget class="QWidget" name="QtWidgetstOperateLogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1876</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgetstOperateLog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>运行日志</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>操作日志</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetOperateLog">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>40</y>
     <width>1851</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>模块名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>机器序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作内容</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作员</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作时间</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableWidgetOperateLog_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>1851</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>解决方案</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>机器序号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>日志内容</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>检验人</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>10</y>
     <width>661</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>开始时间：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="dateEdit">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2023</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>结束时间：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2023</year>
        <month>2</month>
        <day>27</day>
       </datetime>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="pushButtonQuery">
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
